### PR TITLE
Add speaker's dinner page

### PR DESCRIPTION
**Pull request by clytaemnestra**

### CODE REVIEW

File: src/content/pages/speakers-dinner.mdx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+title: Speaker's Dinner
+url: /speakers-dinner
+subtitle:
+  We're happy to invite all speakers to our special Speaker's Dinner!
+---
+
+# Speaker's Dinner
+We are excited to announce a special Speaker's Dinner! This exclusive event is for speakers only, providing an opportunity to meet and mingle before the main conference days.
+
+The dinner will feature multiple courses, and you can choose from a variety of alcoholic and non-alcoholic beverages. We also offer vegetarian, vegan, and gluten-free options to accommodate all dietary preferences.
```
Comment (cybit) [inline src/content/pages/speakers-dinner.mdx:11]:
```suggestion
The dinner will feature multiple courses and you can choose from a variety of alcoholic and non-alcoholic beverages. We also offer vegetarian, vegan, and gluten-free options to accommodate all dietary preferences.
```
I believe that food and drinks should be kept together and therefore the separating "," is not the right choice?

File: src/content/pages/speakers-dinner.mdx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+Address:
+<address>
+Masarykovo nábř. 250/1 <br/>
+110 00 Nové Město, Czechia<br/>
+
+</address>
+
+![](./images/manes.jpg)
+
+## How to arrive there?
```
Comment (cybit) [inline src/content/pages/speakers-dinner.mdx:25]:
```suggestion
## How to get there?
```
Probably nitpicking.

File: src/content/pages/speakers-dinner.mdx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+* Take trams 2, 5, 9, 13, 15, 17, 18, 19, 22, 24, or 27 to Jiráskovo náměstí station, which is a 2-minute walk from the restaurant.
```
Comment (cybit) [inline src/content/pages/speakers-dinner.mdx:26]:
```suggestion
* Take trams 2, 5, 9, 13, 15, 17, 18, 19, 22, 24, or 27 to Jiráskovo náměstí station. From there it is a 2-minute walk to the restaurant.
```

File: src/content/pages/speakers-dinner.mdx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+* [30 minutes on foot](https://www.google.com/maps/dir/Prague+Congress+Centre,+5.+kv%C4%9Btna,+Prague+4-Nusle,+Czechia/Art+Restaurant+M%C3%A1nes,+Masarykovo+n%C3%A1b%C5%99.+250%2F1,+110+00+Nov%C3%A9+M%C4%9Bsto,+Czechia/@50.0697187,14.4116608,15z/data=!4m15!4m14!1m5!1m1!1s0x470b9464c186eb79:0x4d26855708eb61f7!2m2!1d14.4290206!2d50.0625764!1m5!1m1!1s0x470b952eddf2f267:0xbd409c6e6af026e!2m2!1d14.41412!2d50.0774996!3e2!5i2?entry=ttu) from the Prague Congress Center
+
+# When?
+On Tuesday July 9th at 7.30 p.m.
+
+# Registration
+Please note that registration is required. Register through Pretix by adding the event to your list.
```
Comment (cybit) [inline src/content/pages/speakers-dinner.mdx:33]:
```suggestion
Please note that registration is required. Register through Pretix by adding the event to your speaker ticket.
```

File: src/data/links.json [hunk 5]
```diff
@@ -27,6 +27,10 @@
         {
           "name": "Django Girls Workshop",
           "path": "https://djangogirls.org/en/prague/"
+        },
+        {
+          "name": "Speaker's Dinner",
```
Comment (cybit) [inline src/data/links.json:32]:
```suggestion
          "name": "Speakers' Dinner",
```
It is the dinner of the speakers (plural) . So I would guess the apostrophe would go after the "s" ?

File: src/content/pages/speakers-dinner.mdx (hunk 1 above)
Comment (clytaemnestra) [reply src/content/pages/speakers-dinner.mdx:11]: ChatGPT says `,` makes it more readable. 🤷🏻‍♀️ 

image

Comment (hypha) [reply src/content/pages/speakers-dinner.mdx:11]: > vegetarian, vegan, and gluten-free options to accommodate all dietary preferences.

this sentence suggests vegetarian, vegan and gf are all the dietary preferences. 

If you don't have any other option, perhaps something like this is clearer: "We also offer  vegetarian, vegan, and gluten-free options to cater to various dietary preferences."

If you have other options to choose from, maybe "We also offer a variety of options to accommodate different dietary preferences, including vegetarian, vegan, and gluten-free."?

Comment (clytaemnestra) [reply src/content/pages/speakers-dinner.mdx]: Thanks for the suggestion, I've included the second sentence.

File: src/content/pages/speakers-dinner.mdx [hunk 6]
```diff
@@ -0,0 +1,33 @@
+---
+title: Speakers' Dinner
+url: /speakers-dinner
+subtitle:
+  We're happy to invite all speakers to our special Speakers' Dinner!
+---
+
+# Speaker's Dinner
```
Comment (hypha) [inline src/content/pages/speakers-dinner.mdx:8]:
```suggestion
# Speakers' Dinner
```
Just to be consistent throughout.

File: src/content/pages/speakers-dinner.mdx [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+We are excited to announce a special Speakers' Dinner! This exclusive event is for speakers only, providing an opportunity to meet and mingle before the main conference days.
+
+The dinner will feature multiple courses, and you can choose from a variety of alcoholic and non-alcoholic beverages. We also offer a variety of options to accommodate different dietary preferences, including vegetarian, vegan, and gluten-free.
+
+# Where?
+The Speakers' Dinner will be at the Mánes restaurant on the Vltava riverfront in Prague, featuring creative functionalist architecture and panoramic views of the Vltava River, the Dancing House, and the National Theatre.
```
Comment (hypha) [inline src/content/pages/speakers-dinner.mdx:14]: > at the Mánes restaurant
Might it be a good idea to link the URL to the restaurant?

**Review by hypha — COMMENTED** [review]
wee suggestions

File: src/content/pages/speakers-dinner.mdx (hunk 7 above)
Comment (clytaemnestra) [reply src/content/pages/speakers-dinner.mdx]: added ✅

**Comment by cybit** [comment]
Looks good to me.

**Pull request merged**